docker/compose/monitoring/prometheus.yaml

```diff
@@ -0,0 +1,36 @@
+# my global config
+global:
+  scrape_interval:     120s # By default, scrape targets every 15 seconds.
```

Comment:
Why only 2min scrapes? We do 15s for validators.

Reply:
Trying to store less data since this is not for production usage. 😛